gradle.properties

```diff
@@ -36,5 +38,5 @@ POM_DEVELOPER_NAME=Uber Technologies
 
 GITHUB_OWNER=uber
 GITHUB_REPO=rides-java-sdk
-githubDownloadPrefix=https://github.com/uber/rides-java-sdk/releases/download/
+GITHUB_DOWNLOAD_PREFIX=https://github.com/uber/rides-java-sdk/releases/download/
```

Comment:
I noticed that the gradle file in the Android repo escapes the `:`'s, it seems like it doesn't matter, but unsure so wanted to make a note.